department-of-veterans-affairs/caseflow
# CO parent records to Caseflow!

Resolves #7989 

### Description
This PR allows us to create parent records in Caseflow starting January 1st.

### Testing Plan
1. Create a parent record for a CO hearing in Caseflow for January 5th.
2. Go to the View Schedule page and confirm the hearing day is displayed.
3. Go to the Assign Hearings page, schedule a veteran for the parent record, and confirm the record saves successfully, shows up on the daily docket, and shows up on the Scheduled Hearings tab.
